docs/tileentities/tesr.md

```diff
@@ -1,18 +1,18 @@
-TileEntitySpecialRenderer
-=========================
+TileEntityRenderer
+==================
 
-A `TileEntitySpecialRenderer` (TESR) is used to render blocks in a way that cannot be represented with a static baked model (JSON, OBJ, B3D, others). A TESR requires the block to have a TileEntity.
+A `TileEntityRenderer` (colloquially called TESR) is used to render blocks in a way that cannot be represented with a static baked model (JSON, OBJ, B3D, others). A TESR requires the block to have a TileEntity.
```

Comment:
Perhaps `previously TileEntitySpecialRenderer, or TESR for short`. We should be moving away from `TESR` as an abbreviation.